index.d.ts

```diff
@@ -0,0 +1,36 @@
+declare namespace execall {
+	interface Match {
+		match: string;
+		sub: string[];
```

Comment:
```suggestion
		sub: readonly string[];
```

Does it make sense to use `readonly` everywhere? I'm curious.

Reply:
I wouldn't use it on returned values. They should be as flexible and consumer-friendly as possible.